File: WebAppVen/Scripts/app/model/PhMailspM.js
Ext.define('WEBAPP.model.PhMailspM', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'INID', type: 'string' }, 
        { name: 'MGROUP', type: 'string' }, 
        { name: 'MSGNO', type: 'string' },
        { name: 'MSGTEXT', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string'   }
    ]
});

File: WebApp/Models/PH/PH_MAILSP_M.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class PH_MAILSP_M : JCLib.Mvc.BaseModel
    {
        public string INID { get; set; }
        public string MSGRECNO { get; set; }
        public string MSGNO { get; set; }
        public string MSGTEXT { get; set; }
        public string MEMO { get; set; }
        public string TP { get; set; }
        public string M_CONTID { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string DLINE_DT { get; set; }
        public string SMEMO { get; set; }
        public string REDDISP { get; set; }
        public string CONTRACNO { get; set; }        
    }
}